tests/unit/lms/services/jstor/service_test.py

```diff
@@ -1,17 +1,23 @@
 from datetime import timedelta
 from functools import partial
-from unittest.mock import sentinel
+from unittest.mock import patch, sentinel
 
 import pytest
 
 from lms.services import ExternalRequestError
 from lms.services.jstor.service import ArticleNotFound, JSTORService
 from tests import factories
 
-API_URL = "http://api.jstor.org"
+API_URL = "http://jstore_api.example.com"
```

Comment:
Just some belt and braces against making real calls from the tests. Shouldn't happen, but doesn't cost us anything